tests/test_skeleton.py

```diff
@@ -110,16 +110,16 @@ def test_PROPNAME(dash_threaded):
     """Test that some prop updates correctly when changed."""
 
     def assert_callback(
-            component_PROPNAME,
             nclicks,
+            component_PROPNAME,
             input_PROPNAME
     ):
         """Determine the pass/fail status of this test.
 
-        :param (string) component_PROPNAME: The value of PROPNAME for the
-                                            component after it is set
         :param (int) nclicks: The number of clicks on the button in the
                               simple test app (not used here)
+        :param (string) component_PROPNAME: The value of PROPNAME for the
+                                            component after it is set
         :param (string) input_PROPNAME: The value of PROPNAME that is sent
```

Comment:
Small nitpick - if this docstring is a full sentence with a period, should the other ones follow suit?

Reply:
Yes, I originally added the period when editing, then noticed that other param descriptions didn't end with a period, except for the last param... Will add periods everywhere then!

Reply:
Also, should we use the same param description for both `nclicks`?

"The n_clicks value of the button in the simple app." and "The n_clicks value of the button in the simple app (not used here)." instead of "The number of clicks on the button in the simple test app (not used here)."

Reply:
I think that's a good idea!

Reply:
Thanks, done! Meanwhile, I have also 'fixed' the order argument name/type.